src/js/components/Data/index.d.ts

```diff
@@ -104,6 +104,8 @@ export interface DataProps {
       order?: string;
       select?: string;
       tip?: string;
+      orderAria?: string;
```

Comment:
For SelectMultiple, we use the term a11y instead of aria for the aria specific label -- do we want to standardize on one direction moving forward?

aka if we prefer "aria", I'm good with this.

Reply:
I feel like if we already use `a11y` we could standardize to that